openc3-init/plugins/packages/openc3-tool-common/src/plugins/dialog/ConfirmDialog.vue

```diff
@@ -27,6 +20,13 @@
           @click="cancel"
           >{{ params.cancelText }}</v-btn
         >
+        <v-btn
+          class="mx-2"
+          color="primary"
+          :data-test="dataTestOk"
+          @click="ok"
+          >{{ params.okText }}</v-btn
+        >
```

Comment:
Material design says to have the primary action button on the right. We should eventually check all the other dialogs for consistency.